server/tests/unit/common/config/test_server_config.py

```diff
@@ -47,7 +47,7 @@ def test_init_raises_error_if_default_config_is_invalid(self):
     def test_complete_config_checks_all_attr(self, mock_check_attrs):
         mock_check_attrs.side_effect = BaseConfig.validate_correct_type_of_configuration_attribute()
         self.server_config.complete_config(self.context)
-        self.assertEqual(mock_check_attrs.call_count, 36)
+        self.assertEqual(mock_check_attrs.call_count, 32)
```

Comment:
There are now 4 less config properties related to removal of caching, so this is a valid test update